websocket/controller.cc

```diff
@@ -35,16 +27,34 @@ future<> controller::start_server()
     }
     _active = true;
     co_await _server.start();
-    co_await _server.invoke_on_all([] (experimental::websocket::server& ws) {
-        ws.register_handler("echo", [] (input_stream<char>& in, output_stream<char>& out) -> future<> {
-            while (true) {
-                temporary_buffer<char> f = co_await in.read();
-                if (f.empty()) {
-                    co_return;
```

Comment:
I think adding this was pretty pointless, better add the barebone infra instead first, adding meaningful handlers in later commits.